docs/starting-from-scratch.md

```diff
@@ -1,142 +1,449 @@
 
-Check out [this blog post]() to learn more.   
+For this tutorial, we will be focused on creating a one code repo, one build single-spa application from scratch using React and Angular1(AngularJS) with Webpack. 
 
-## Step One: choose a module loader.
-Your module loader / bundler is the library you’ll use to [lazy load](https://webpack.js.org/guides/lazy-loading/) code. I recommend either [Webpack](https://webpack.js.org/) or [JSPM](https://jspm.io/), if you’re starting from scratch.
+It is important to note that you do not need to use webpack to use single-spa. 
+<!-- Would love some guidance here -->
+It is simply an option that ...........? 
 
-If you go with Webpack, try to use Webpack 2 if you can, since it has support for promise-based [lazy loading](https://webpack.js.org/guides/lazy-loading/). This will make things easier for you later on, since single-spa requires that your loading functions return promises. If you can’t use Webpack 2, getting single-spa to lazy load your code with Webpack 1 will require some boilerplate code.
+You can find the completed [project repo here](). Read more about [seperating applications](https://single-spa.js.org/docs/separating-applications.html) using single-spa. 
 
-JSPM/SystemJS has worse documentation than Webpack, but is a great solution for module loading if you can get past that. I recommend using jspm@0.17 — it’s still in beta but has been worked on for over a year and at Canopy we find it stable enough to use in production.
+Be sure to read through the [singe-spa docs](https://single-spa.js.org/), check out the other tutorials and the [single-spa github](https://github.com/CanopyTax/single-spa) for more info.
 
-If you’re struggling to decide between the two, then ask yourself the following: Do I want multiple completely separate bundles? If you don’t, I recommend Webpack because it has better docs, a larger community, and fewer gotchas. Otherwise, I’d go with JSPM, since Webpack has no plans to support dynamic runtime loading. 
+## Step One: Project Set up 
 
-Here is an example of how you can set up your [webpack.config](https://github.com/joeldenning/simple-single-spa-webpack-example/blob/master/webpack.config.js) file.
+Create a new directory to house our project, followed by a `src` folder to hold all of our micro-service applications.
 
-## Step Two: create a brand new html file
-The next step is to create what single-spa calls your “[root application](https://github.com/CanopyTax/single-spa/blob/master/docs/root-application.md).” Really your root application is just the stuff that initializes single-spa, and it starts with an html file.
+```bash
+mkdir single-spa-simple-example && cd single-spa-simple-example
+mkdir src
+```
+
+In the root of our new project, initialize the package manager of your choice and install single-spa. For this tutorial, we will be using [yarn](https://yarnpkg.com/en/). 
```

Comment:
Really there are only two package managers worth noting here: `npm` and `yarn`. `jspm` is the only other one I am aware of, but it is not very popular and anyone using it will know there way around the javascript ecosystem haha.

I might change this to say the following:
>Next, initialize your project and install single-spa as a dependency:
> ```bash
> yarn init              # or npm init
> yarn add single-spa    # or npm install --save single-spa
> ```